### PR TITLE
Release 2.21.1 (develop)

### PR DESCRIPTION
**Pull request by Adamj1232**
### **✨ Highlights**

- Update WalletConnect eth_RequestAccounts flow along with update WC deps and add new optional params #1622 
- Why Don't I See My Wallet Link #1617 

Docs:
- Update Notify section on the docs site and within readme #1612 
- Docs Release Favicon Update #1613 
- Fix broken link in docs #1616 
- Update Broken Links on Docs Pages #1619 

📦 **Changes per package**
- @web3-onboard/core: 2.16.1
- @web3-onboard/transaction-preview: 2.0.5
- @web3-onboard/react: 2.7.1
- @web3-onboard/walletconnect: 2.3.4
- @web3-onboard/vue: 2.6.1

### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
   "type": "module",
   "dependencies": {
     "@web3-onboard/coinbase": "^2.2.1",
-    "@web3-onboard/core": "^2.16.1-alpha.2",
+    "@web3-onboard/core": "^2.16.0",
```
Comment (leightkt) [inline docs/package.json:58]: Should this be 2.16.1?

File: docs/package.json [hunk 2]
```diff
@@ -76,7 +76,7 @@
     "@web3-onboard/trezor": "^2.3.4",
     "@web3-onboard/trust": "^2.0.1",
     "@web3-onboard/uauth": "^2.0.2",
-    "@web3-onboard/walletconnect": "^2.3.4-alpha.1",
+    "@web3-onboard/walletconnect": "^2.3.3",
```
Comment (leightkt) [inline docs/package.json:79]: 2.3.4?

**Review by leightkt — APPROVED** [review]
A couple of version questions in the docs/package.json (which may just be me confused on versioning again)
otherwise- looking good!

File: docs/package.json (hunk 1 above)
Comment (Adamj1232) [inline docs/package.json:58]:
```suggestion
    "@web3-onboard/core": "^2.16.1",
```

File: docs/package.json (hunk 2 above)
Comment (Adamj1232) [inline docs/package.json:79]:
```suggestion
    "@web3-onboard/walletconnect": "^2.3.4",
```

**Pull request merged**